attache.gemspec

```diff
@@ -0,0 +1,37 @@
+$:.push File.expand_path("../lib", __FILE__)
+
+# Maintain your gem's version:
+require "attache/version"
+
+# Describe your gem and declare its dependencies:
+Gem::Specification.new do |s|
+  s.name        = "attache"
+  s.version     = Attache::VERSION
+  s.authors     = ["choonkeat"]
+  s.email       = ["choonkeat@gmail.com"]
+  s.homepage    = "https://github.com/choonkeat/attache"
+  s.summary     = "Image server for everybody"
+  s.description = "Standalone rack app to manage files onbehalf of your app"
+  s.license     = "MIT"
+
+  s.files       = Dir["{app,lib}/**/*", "MIT-LICENSE", "Rakefile", "README.md", 'exe/**/*']
+  s.bindir      = 'exe'
+  s.executables = ['attache']
+
+  s.add_runtime_dependency 'rack', '~> 1.6'
+  s.add_runtime_dependency 'paperclip', '~> 4.3'
+  s.add_runtime_dependency 'puma', '~> 2.14'
+  s.add_runtime_dependency 'fog', '~> 1.34'
+  s.add_runtime_dependency 'excon', '~> 0.45'
+  s.add_runtime_dependency 'sys-filesystem', '~> 0'
+  s.add_runtime_dependency 'disk_store', '~> 0'
+  s.add_runtime_dependency 'celluloid', '< 0.17' # 0.17 has compatibility issues with disk_store
+  s.add_runtime_dependency 'foreman', '~> 0'
```

Comment:
@choonkeat The author of foreman [suggests](https://github.com/ddollar/foreman/pull/486#issuecomment-62082372) user install foreman to the system by themselves.